extensions/wikia/AppPromoLanding/AppPromoLandingController.class.php

```diff
@@ -224,11 +190,10 @@ public function content( $params ){
 	 *         there is no app configured for the given wiki, then null is returned.
 	 */
 	static private function getConfigForWiki( $cityId ){
-		wfProfileIn( __METHOD__ );
 		$desiredConfig = null;
 
 		// Gets the configs for ALL apps.
-		$appConfig = AppPromoLandingController::getAllAppConfigs();
+		$appConfig = $this->getAllAppConfigs();
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Remove this use of "$this". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2014)